From 94bba6880b1f10c6b3bf33a17ac40935d65a81ae Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Fri, 6 Nov 2015 15:19:46 +0000
Subject: [PATCH] Don't remove the system libraries and startup files from
 libaio, as in some build configurations these are required.  For example,
 including conf/include/security_flags.inc on PPC results in:

io_queue_init.os: In function `io_queue_init':
tmp/work/ppce300c3-poky-linux/libaio/0.3.110-r0/libaio-0.3.110/src/io_queue_init.c:33:
undefined reference to `__stack_chk_fail_local'

Upstream-Status: Pending
Signed-off-by: Ross Burton <ross.burton@intel.com>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 37ae219..22e0c9a 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -6,7 +6,7 @@ CFLAGS ?= -g -fomit-frame-pointer -O2
 CFLAGS += -Wall -I. -fPIC
 SO_CFLAGS=-shared $(CFLAGS)
 L_CFLAGS=$(CFLAGS)
-LINK_FLAGS=
+LINK_FLAGS=$(LDFLAGS)
 LINK_FLAGS+=$(LDFLAGS)
 ENABLE_SHARED ?= 1
 
-- 
2.25.1

